target/linux/realtek/image/rtl838x.mk

```diff
@@ -39,6 +42,11 @@ TARGET_DEVICES += d-link_dgs-1210-20
 define Device/d-link_dgs-1210-28
   $(Device/d-link_dgs-1210)
   DEVICE_MODEL := DGS-1210-28
+  FACTORY_IMG_CUSTOMER_SIGNATURE := 2
+  FACTORY_IMG_BOARD_VERSION := 32
+  IMAGES += factory_only_for_image1.bin
+  IMAGE/factory_only_for_image1.bin := append-kernel | pad-to 64k | append-rootfs | pad-rootfs | \
+	          pad-to 16 | check-size | dlink-header
```

Comment:
`dlink-header` will make the file bigger, no? Unless the vendor FW will unpack the blobs and the padded file is to one actually written to flash (not clear to me from the commit message), `dlink-header` must be called before `check-size`.

Reply:
It drops the headers before flashing. IMAGE_SIZE is exactly the sum of the kernel and the rootfs size. I'll try to explain it better in the commit message.

Reply:
Added some extra info to commit message